OSDOCS-5196: Updated 7.0 version in VSphere docs

dfitzmau opened the pull request:
Issue:
[OSDOCS-5196](https://issues.redhat.com/browse/OSDOCS-5197)

Version(s):
4.3

Link to docs preview:
<!--- Add direct link(s) to the exact page(s) with updated content from the preview build. --->

QE review:
- [ ] QE has approved this change.
<!--- QE approval is required to merge a PR except for changes that do not impact the meaning of the docs. --->

dfitzmau commented:
Hi @rvanderp3 .

I updated the version number as per the docs update request in [OSDOCS-5196](https://issues.redhat.com/browse/OSDOCS-5197). Please let me know if these updates capture what you requested. I added preview links at the start of this PR that each show the changes. 

I see the following note in the docs that might also require update to the version numbers? However, `7.0 update 0` sounds a bit off for some reason (as in it's not an update). Please let me know your thoughts. 

![Screenshot from 2023-02-14 13-12-17](https://user-images.githubusercontent.com/57954076/218749097-513167d0-9682-476d-a7f3-a42695b56c4d.png)

rvanderp3 commented:
> Issue: [OSDOCS-5196](https://issues.redhat.com/browse/OSDOCS-5197)
> 
> Version(s): 4.3
> 
> Link to docs preview:
> 
> * [Preparing to install on vSphere](https://55872--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/preparing-to-install-on-vsphere.html)
> * [Installing a cluster on vSphere](https://55872--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned.html)
> 
> QE review:
> 
> * [ ]  QE has approved this change.

I think you mean 4.13 here

rvanderp3 commented:
> Hi @rvanderp3 .
> 
> I updated the version number as per the docs update request in [OSDOCS-5196](https://issues.redhat.com/browse/OSDOCS-5197). Please let me know if these updates capture what you requested. I added preview links at the start of this PR that each show the changes.
> 
> I see the following note in the docs that might also require update to the version numbers? However, `7.0 update 0` sounds a bit off for some reason (as in it's not an update). Please let me know your thoughts.
> 
> ![Screenshot from 2023-02-14 13-12-17](https://user-images.githubusercontent.com/57954076/218749097-513167d0-9682-476d-a7f3-a42695b56c4d.png)

@jcpowermac 7.0 and 7.0u1 aren't just deprecated are they?  You must be running 7.0u2 or later, right?

rvanderp3 commented:
> > Hi @rvanderp3 .
> > I updated the version number as per the docs update request in [OSDOCS-5196](https://issues.redhat.com/browse/OSDOCS-5197). Please let me know if these updates capture what you requested. I added preview links at the start of this PR that each show the changes.
> > I see the following note in the docs that might also require update to the version numbers? However, `7.0 update 0` sounds a bit off for some reason (as in it's not an update). Please let me know your thoughts.
> > ![Screenshot from 2023-02-14 13-12-17](https://user-images.githubusercontent.com/57954076/218749097-513167d0-9682-476d-a7f3-a42695b56c4d.png)
> 
> @jcpowermac 7.0 and 7.0u1 aren't just deprecated are they? You must be running 7.0u2 or later, right?

though i guess i'm not clear on which version this PR is targeting, for 4.12, deprecated may be the right language.

jcpowermac commented:
I think we should have PM comment @racedo

dfitzmau commented:
> > Issue: [OSDOCS-5196](https://issues.redhat.com/browse/OSDOCS-5197)
> > Version(s): 4.3
> > Link to docs preview:
> > 
> > * [Preparing to install on vSphere](https://55872--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/preparing-to-install-on-vsphere.html)
> > * [Installing a cluster on vSphere](https://55872--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned.html)
> > 
> > QE review:
> > 
> > * [ ]   QE has approved this change.
> 
> I think you mean 4.13 here

Yes. Thanks. Well spotted!

dfitzmau commented:
> > > Hi @rvanderp3 .
> > > I updated the version number as per the docs update request in [OSDOCS-5196](https://issues.redhat.com/browse/OSDOCS-5197). Please let me know if these updates capture what you requested. I added preview links at the start of this PR that each show the changes.
> > > I see the following note in the docs that might also require update to the version numbers? However, `7.0 update 0` sounds a bit off for some reason (as in it's not an update). Please let me know your thoughts.
> > > ![Screenshot from 2023-02-14 13-12-17](https://user-images.githubusercontent.com/57954076/218749097-513167d0-9682-476d-a7f3-a42695b56c4d.png)
> > 
> > 
> > @jcpowermac 7.0 and 7.0u1 aren't just deprecated are they? You must be running 7.0u2 or later, right?
> 
> though i guess i'm not clear on which version this PR is targeting, for 4.12, deprecated may be the right language.

Hi @rvanderp3 . This PR targets the 4.13 release, so deprecated might still be OK? 

Version numbers should state 7.0 update 0 and 7.0 update 1? If not, why the change for 7.0 update 2? We usually try to maintain consistency among terms, versions, and so on; this avoids distracting content that can cause a doc's user to lose their focus.

racedo commented:
This is to ensure that *OpenShift 4.13* on vSphere is limited to vSphere 7.0 Update 2 or later for installations and upgrades, because OCP 4.13 will have CSI migration enabled and vSphere 7.0 Update 2 is the minimum version that meets this requirement of migration. 

Earlier vSphere versions are not supported in OCP 4.13.

Upgrades to OpenShift 4.13 will be blocked if vSphere 7.0 Update 1 or earlier is used. vSphere 7.0 Update 2 or later is the minimum version required for upgrade.

Also, vSphere 7.0 update 0 isn't referenced like that anywhere, it's simply vSphere 7.0. Using "Update"  (note vmware users uppercase  U) starts in "Update 1".

dfitzmau commented:
Thanks, @racedo , for the comments. I updated the version number references according to your feedback. 

I can see your point about the deprecated notice. I'm also think the notice does not apply well to the strict 7.0 Update 2 or later versions for 4.13. The notice should mention that 7.0 and 7.0 Update 1 are not compatible with installing OCP 4.13 on vSphere?

dfitzmau commented:
I also see this paragraph in the pre-existing text:

> You must install the {product-title} cluster on a VMware vSphere version 7.0 Update 2 or later instance that meets the requirements for the components that you use.

Cluster is mentioned here also, but maybe it should state the high-level OCP entity? This would help clarify your last comment along with an update to the notice (remove the cluster reference)?

rvanderp3 commented:
LGTM
cc: @jcpowermac

duanwei33 commented on modules/installation-vsphere-infrastructure.adoc:
```diff
@@ -45,39 +45,43 @@ endif::[]
 [IMPORTANT]
 ====
-Installing a cluster on VMware vSphere versions 7.0.0 and 7.0.1 is deprecated. These versions are still fully supported, but all vSphere 6.x versions are no longer supported. Version 4.12 of {product-title} requires VMware virtual hardware version 15 or later. To update the hardware version for your vSphere virtual machines, see the "Updating hardware on nodes running in vSphere" article in the _ ...
+Installing a cluster on VMware vSphere versions 7.0 and 7.0 Update 1 is deprecated. These versions are still fully supported, but all vSphere 6.x versions are no longer supported. Version 4.12 of {product-title} requires VMware virtual hardware version 15 or later. To update the hardware version for your vSphere virtual machines, see the "Updating hardware on nodes running in vSphere" article in  ...
```
> Installing a cluster on VMware vSphere versions 7.0 and 7.0 Update 1 is deprecated. These versions are still fully supported, but all vSphere 6.x versions are no longer supported.

I understand we don't support installation on 7.0.1 in 4.13, right?



> Version 4.12 of {product-title} requires VMware virtual hardware version 15 or later

And we should mention 4.13 here.

cc @sgaoshang

dfitzmau replied:
Thanks, @duanwei33 . I updated the content to state 4.13.

I'll await @sgaoshang response on dropping the 7.0 and 7.0 Update 1 sentence.

sgaoshang replied:
I'm also prefer we drop the 7.0 and 7.0 Update 1 sentence, thanks.

jcpowermac replied:
the 4.13 installer will not allow below vSphere 7 U2 components...that includes *all* esxi hosts in the defined clusters and vcenter.

dfitzmau replied:
Thank you, all. I removed this admonition from the text.

dfitzmau commented:
/label peer-review-needed

mburke5678 commented on modules/installation-vsphere-infrastructure.adoc:
```diff
@@ -45,39 +45,38 @@ endif::[]
 
+[NOTE]
+====
+Version 4.13 of {product-title} supports VMware vSphere version 8.0.
```
s/Version 4.13 of {product-title}/ {product-title} version 4.13

dfitzmau commented:
Thanks, @mburke5678 . I updated the content based on your suggestion. Please let me know your thoughts.

dfitzmau commented:
/label merge-review-needed

ousleyp commented:
LGTM. Merging shortly!

ousleyp commented:
/cherrypick enterprise-4.13
